UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/Models/D365Payload.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace UKHO.ExternalNotificationService.API.Models
+{
+    public class D365Payload 
+    {
+        public string CorrelationId { get; set; }
+        public string OperationCreatedOn { get; set; }
+        public EntityImage[] PostEntityImages { get; set; }
+        public InputParameter[] InputParameters { get; set; }
+    }
+
+    public class InputParameter
+    {
+        public InputParameterValue value { get; set; }
```

Comment:
Please maintain style consistency in the code for all `key` & `value` properties in this file by applying the `JsonPropertyNameAttribute` like this:
```suggestion
        [JsonPropertyName("value")]
        public InputParameterValue Value { get; set; }
```